src/api/app/views/webui/configuration/index.html.haml

```diff
@@ -5,6 +5,14 @@
   = render partial: 'tabs'
   .card-body
     %h3 Configure your Open Build Service instance
+    %p
+      Settings that aren't available here are only available by modifying the
+      %code config/options.yml
+      file for the frontend and
+      %code BSConfig.pm
+      for the backend.
+    - inherit = sanitize(['The value for <code>%s</code> is already set in a configuration file. ',
+                          "Remove it from <code>config/options.yml</code> in order to modify it here."].join)
```

Comment:
This assignment could be moved to the beginning of the file, under the `:ruby` code section. .

```suggestion
    inherit = sanitize('The value for <code>%s</code> is already set in a configuration file. ' +
      "Remove it from <code>config/options.yml</code> in order to modify it here.")
```